gst/rtp/gstrtph264depay.c

```diff
@@ -1260,3 +1333,24 @@ gst_rtp_h264_depay_plugin_init (GstPlugin * plugin)
   return gst_element_register (plugin, "rtph264depay",
       GST_RANK_SECONDARY, GST_TYPE_RTP_H264_DEPAY);
 }
+
+static gboolean
+gst_rtp_h264_depay_packet_lost (GstRTPBaseDepayload * depay, GstEvent * event)
```

Comment:
> and be sure to call the base class method from yours

https://developer.gnome.org/gobject/stable/howto-gobject-chainup.html

By overriding the base class' meethod with your implementation like this you stop sending the gap event.

Reply:
fixed